core/pom.xml

```diff
@@ -76,6 +76,18 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.kafka</groupId>
```

Comment:
Doesn't this already get pulled in via the `kafka_${kafka.scala.version}` dependency?